lib/codit/population/person.py

```diff
@@ -40,13 +40,13 @@ def attack(self, other, days):
             self.infectious_attack(other, days)
 
     def infectious_attack(self, other, days):
-        if not other.infected:
+        if self.disease not in other.immunities:
```

Comment:
This is the one place that will remain `.immunities` - everywhere else becomes `.covid_experiences`

Reply:
When we get to vacciations, hopefully soon, we probably won't want to have an in/out criterion here, but a softer one which adjusts `self.disease.pr_transmit_per_day` downwards by a configurable ratio depending on `other.immunities`